### ISSUE TITLE
Hide Stack in Tab's with @react-navigation/material-bottom-tabs

### ISSUE THREAD

**Issue by astrahov** [react-navigation/react-navigation#7863]
**Current Behavior**

```
import { AppRegistry } from 'react-native'
import { name as appName } from './app.json'

import 'react-native-gesture-handler'
import React from 'react'
import { NavigationContainer } from '@react-navigation/native'
import { enableScreens } from 'react-native-screens'
import { Provider as PaperProvider } from 'react-native-paper'

import { Platform } from 'react-native'
import { createBottomTabNavigator } from '@react-navigation/bottom-tabs'
import { createStackNavigator } from '@react-navigation/stack'
import { createMaterialBottomTabNavigator } from '@react-navigation/material-bottom-tabs'

enableScreens()



import FirstScreen from '@/screens/Test/First/FirstScreen'
import SecondScreen from '@/screens/Test/Second/SecondScreen'
import ThirdScreen from '@/screens/Test/Third/ThirdScreen'

const FirstStack = createStackNavigator()

function FirstStackScreen() {
    return (
        <FirstStack.Navigator initialRouteName="First">
            <FirstStack.Screen name="First" component={FirstScreen} />
        </FirstStack.Navigator>
    )
}

const SecondStack = createStackNavigator()

function SecondStackScreen() {
    return (
        <SecondStack.Navigator initialRouteName="Second">
            <SecondStack.Screen name="Second" component={SecondScreen} />
        </SecondStack.Navigator>
    )
}

const ThirdStack = createStackNavigator()

function ThirdStackScreen() {
    return (
        <ThirdStack.Navigator initialRouteName="Third">
            <ThirdStack.Screen name="Third" component={ThirdScreen} />
        </ThirdStack.Navigator>
    )
}

function RootStackScreen() {
    return (
        <HomeTab.Navigator
            initialRouteName="First"
            screenOptions={({ route }) => ({
                headerShown: false,
            })}>
            <HomeTab.Screen name="First" component={FirstScreen} />
            <HomeTab.Screen name="Second" component={SecondStackScreen} />
            <HomeTab.Screen name="Third" component={ThirdStackScreen} />
        </HomeTab.Navigator>
    )
}

class App extends React.Component {
    render() {
        return (
            <PaperProvider>
                <NavigationContainer>
                    <RootStackScreen />
                </NavigationContainer>
            </PaperProvider>
        )
    }
}

AppRegistry.registerComponent(appName, () => App)
```

!!! Only Android

When switching between tabs within which Stack Navigators - an error occurs - the screen does not appear inside the Stack.

[video](https://drive.google.com/open?id=17xGmqBkAMA3ghrSWqFTrGlz2WMwawJs5)

------------

**Expected Behavior**

When switching between tabs within which Stack Navigators - Stack and the screen inside it should open

**How to reproduce**

!!! Only Android

1. Create a TabNavigator
2. Create 3 StackNavigator's with a screen inside
3. Switch between tabs

**Your Environment**

| software                               | version |
| -------------------------------------- | ------- |
| Android                         | 9.0 Pie
| @react-navigation/native               | 5.1.3
| @react-navigation/material-bottom-tabs | 5.1.6
| react-native-paper                     | 3.6.0
| react-native                           | 0.61.5
| node                                   | 13.8.0
| npm                            | 6.14.3

**Comment by astrahov** [comment]
Update Environment:

| software                               | version |
| -------------------------------------- | ------- |
| Android                         | 9.0 Pie
| @react-navigation/native               | 5.1.5
| @react-navigation/material-bottom-tabs | 5.1.8
| react-native-paper                     | 3.8.0
| react-native                           | 0.62.2
| node                                   | 13.8.0
| npm                            | 6.14.4

Problem still exists

**Comment by astrahov** [comment]
Update Environment:

| software                               | version |
| -------------------------------------- | ------- |
| Android                         | 9.0 Pie
| @react-navigation/material-bottom-tabs | 5.1.9
| @react-navigation/native | 5.1.6
| @react-navigation/stack | 5.2.11
| react-native-paper                     | 3.8.0
| react-native                           | 0.62.2
| node                                   | 13.8.0
| npm                            | 6.14.4

Problem still exists

**Comment by AnthonyDugarte** [comment]
I am having the same issue, with both material-bottom-tabs and material-top-tabs.

**Comment by raajnadar** [comment]
```
function RootStackScreen() {
    return (
        <HomeTab.Navigator
            initialRouteName="First"
            screenOptions={({ route }) => ({
                headerShown: false,
            })}>
            <HomeTab.Screen name="First" component={FirstScreen} /> // Here is the mistake
            <HomeTab.Screen name="Second" component={SecondStackScreen} />
            <HomeTab.Screen name="Third" component={ThirdStackScreen} />
        </HomeTab.Navigator>
    )
}
```

`FirstScreen` should be `FirstStackScreen`

**Comment by Stringsaeed** [comment]
@astrahov solved with **`react-native-screens@2.7.0`**
